src/mono/mono/metadata/assembly-internals.h

```diff
@@ -120,7 +120,7 @@ MonoAssembly*          mono_assembly_request_load_from (MonoImage *image, const
 							const MonoAssemblyLoadRequest *req,
 							MonoImageOpenStatus *status);
 
-MonoAssembly*          mono_assembly_request_byname (MonoAssemblyName *aname,
+MONO_COMPONENT_API MonoAssembly* mono_assembly_request_byname (MonoAssemblyName *aname,
```

Comment:
What does MONO_COMPONENT_API actually do?

Reply:
Export the function using `__declspec(dllexport)` or `__attribute__ ((__visibility__ ("default")))`